lib/unifex.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Unifex do
```

Comment:
I would place it under `lib/unifex/precompiler.ex` and name this module as `defmodule Unifex.Precompiler` but up to you

Reply:
That way one would have to put `Unifex.Precompiler` there:
https://github.com/membraneframework/unifex/pull/49/files#diff-d4447cfb5dbff840134840900481468bR32

Reply:
Oh, I see :+1: